concept/type/attributetype.feature

```diff
@@ -98,285 +120,119 @@ Feature: Concept Attribute Type
       | name |
       | age  |
     When transaction commits
-    When connection opens read transaction for database: typedb
+    When connection open read transaction for database: typedb
     Then attribute(name) does not exist
     Then attribute(age) does not exist
     Then attribute(attribute) get subtypes do not contain:
       | name |
       | age  |
+    Examples:
+      | value-type-1 | value-type-2 |
+      | string       | long         |
+      | boolean      | double       |
+      | decimal      | datetimetz   |
+      | datetime     | duration     |
 
-  Scenario: Attribute types that have instances cannot be deleted
-    When put attribute type: name
-    When attribute(name) set value-type: string
-    When transaction commits
-    When connection opens write transaction for database: typedb
-    When $x = attribute(name) put: alice
+  Scenario: Attribute types can get the root type
+    When create attribute type: is-open
+    When attribute(is-open) set value type: boolean
+    When create attribute type: age
+    When attribute(age) set value type: long
+    When create attribute type: rating
+    When attribute(rating) set value type: double
+    When create attribute type: name
+    When attribute(name) set value type: string
+    When create attribute type: timestamp
+    When attribute(timestamp) set value type: datetime
+    Then attribute(is-open) get supertype: attribute
+    Then attribute(age) get supertype: attribute
+    Then attribute(rating) get supertype: attribute
+    Then attribute(name) get supertype: attribute
+    Then attribute(timestamp) get supertype: attribute
     When transaction commits
-    When connection opens schema transaction for database: typedb
-    Then delete attribute type: name; fails
+    When connection open read transaction for database: typedb
+    Then attribute(is-open) get supertype: attribute
+    Then attribute(age) get supertype: attribute
+    Then attribute(rating) get supertype: attribute
+    Then attribute(name) get supertype: attribute
+    Then attribute(timestamp) get supertype: attribute
+
+    # TODO: Move to thing-feature or schema/data-validation?
+#  Scenario: Attribute types that have instances cannot be deleted
+#    When create attribute type: name
+#    When attribute(name) set value type: string
+#    When transaction commits
+#    When connection open write transaction for database: typedb
+#    When $x = attribute(name) put: alice
+#    When transaction commits
+#    When connection open schema transaction for database: typedb
+#    Then delete attribute type: name; fails
```

Comment:
I would not do it right now, I'll have a separate PR with `validation` refactoring and it will affect these files as well (some of the `transaction commits; fails` can be reimagined). It's easier to separate my new tests from old validation tests now, it won't be forgotten as I'll clean all the TODOs in our tests, ignoring only some unimplemented features.